airsonic-main/src/test/java/org/airsonic/player/service/MediaScannerServiceTestCase.java

```diff
@@ -161,12 +152,11 @@ public void testScanLibrary() {
 
     @Test
     public void testSpecialCharactersInFilename() throws Exception {
-        Resource resource = resourceLoader.getResource("MEDIAS/piano.mp3");
         String directoryName = "Muff1nman\u2019s \uFF0FMusic";
         String fileName = "Muff1nman\u2019s\uFF0FPiano.mp3";
         File artistDir = temporaryFolder.newFolder(directoryName);
         File musicFile = artistDir.toPath().resolve(fileName).toFile();
-        IOUtils.copy(resource.getInputStream(), new FileOutputStream(musicFile));
+        Resources.copy(Resources.getResource("MEDIAS/piano.mp3"), new FileOutputStream(musicFile));
```

Comment:
This is marked as unstable, I'm not sure we should use it :/

Reply:
This is a guava class, so we should be fine. I think it's just annotated as @Beta, which just means it's not feature-frozen at the class-level.

The Beta javadoc for Guava notes should tell you the meaning of the annotation.